### ISSUE TITLE
Article search not working: flexsearch

### ISSUE THREAD

**Issue by nemu1113** [kwchang0831/svelte-QWER#35]
### Describe the bug

A console error has occurred at the time of the first access, 
and the site search function is not working properly.

### Reproduction

1. Execute "pnpm i"
2. Execute "pnpm husky"
3. Execute "pnpm dev"
4. Access the following URL from browser
http://localhost:5173/

### Logs

```shell
Uncaught (in promise) TypeError: flexsearch.Index is not a constructor
    at Object._init [as init] (index.ts:11:14)
    at search.ts?type=module&worker_file:25:5
```

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (12) x64 AMD Ryzen 5 1600 Six-Core Processor
    Memory: 8.57 GB / 15.92 GB
  Binaries:
    Node: 16.17.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.18.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (105.0.1343.53)    
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @sveltejs/adapter-auto: ^1.0.0-next.80 => 1.0.0-next.80      
    @sveltejs/adapter-node: ^1.0.0-next.96 => 1.0.0-next.96      
    @sveltejs/adapter-static: ^1.0.0-next.44 => 1.0.0-next.44    
    @sveltejs/kit: ^1.0.0-next.509 => 1.0.0-next.509 
    svelte: ^3.50.1 => 3.50.1
    vite: ^3.1.4 => 3.1.4
```

### Additional Information

Enabled the following settings, and url or content etc contain non-ascii character.

`user\config\QWER.config.js`

```js
RoutingRules: {
    PostsYearMonthDate: true,
},
```

**Comment by kwchang0831** [comment]
I can not reprouce this issue.
https://qwer-nemunote-phi.vercel.app 

Need more info...

**Comment by kwchang0831** [comment]
I cannot reproduce in local env neither.

Try removing all node_modules and reinstall again.

```sh
rm node_modules QWER/node_modules -rf && pnpm i
```

**Comment by kwchang0831** [comment]
Thanks for your patience.

This issue should now be resolved as of 2e0a0451432837f14c7c9c4b7b06d6748fd2a5d2.
